website/content/docs/configuration/consul.mdx

```diff
@@ -200,9 +200,10 @@ consul {
 ### Workload Identity
 
 The `service_identity` and `task_identity` blocks accept all the same values as
-the job specification's [`identity`][identity_block]. By ensuring these blocks
-are set on the Nomad servers, you can automatically migrate your existing jobs
-to use Workload Identity without modifying the job specification and
+the job specification's [`identity`][identity_block] (except that the
```

Comment:
This reminds me that we need to update https://github.com/hashicorp/nomad/blob/main/website/content/docs/job-specification/identity.mdx with the new fields 😅

Reply:
Oh, I missed that we hadn't. I'll do that in a follow-up PR.

Reply:
Done in https://github.com/hashicorp/nomad/pull/18916